src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerTeamControllerTests/WhenCallToActionToggleIsEnabled/WhenFundsToReserve.cs

```diff
@@ -68,29 +68,5 @@ public void ThenForNonLevyTheCheckFundingViewIsReturnedAtRow1Panel1()
             Assert.AreEqual("CheckFunding", (result.Model as dynamic).ViewName);
             Assert.AreEqual(PanelType.Interruption, (result.Model as dynamic).PanelType);
         }
-
-        [Test]
```

Comment:
Why has this test been deleted? It is still a relevant state of the Call to Action

Reply:
I have moved to another file becz ' WHEN' condition not applicable for this test